packages/ses/SECURITY.md

```diff
@@ -10,16 +10,34 @@ be made for the most recent branch only.
 | ------- | ------------------ |
 | 0.12.x  | :white_check_mark: |
 
-## Reporting a Vulnerability
-
-Please help us practice coordinated security bug disclosure. If you find a
-security-sensitive bug that should not be revealed publically until a fix is
-available, please send email to `security` at (@) `agoric.com`. To encrypt,
-please use my (@warner) personal GPG key [A476E2E6 11880C98 5B3C3A39 0386E81B
-11CAA07A](http://www.lothar.com/warner-gpg.html) . Keybase users can also
-send messages to `@agoric_security`, or share code and other log files via
-the Keybase encrypted file system
-(`/keybase/private/agoric_security,$YOURNAME`).
-
-We will create a GitHub security advisory promptly. Let us know your GitHub
-username and we'll add you to the collaborator list for further discussion.
+
+## Coordinated Vulnerability Disclosure of Security Bugs
+
+SES stands for fearless cooperation, and strong security requires strong collaboration with security researchers. If you believe that you have found a security sensitive bug that should not be disclosed until a fix has been made available, we encourage you to report it. To report a bug in HardenedJS, you have several options that include: 
+
+* Reporting the issue to the [Agoric HackerOne vulnerability rewards program](hackerone.com/agoric). 
+
+* Sending an email to security at (@) agoric.com., encrypted or unencrypted. To encrypt, please use  @Warner’s personal GPG key  [A476E2E6 11880C98 5B3C3A39 0386E81B 11CAA07A](http://www.lothar.com/warner-gpg.html)  .
+
+* Sending a message on Keybase to `@agoric_security`, or sharing code and other log files via Keybase’s encrypted file system. ((_keybase_private/agoric_security,$YOURNAME).
```

Comment:
((_keybase_private/agoric_security,$YOURNAME)

Is this a left over of the HackerOne template?

Reply:
Yes, kind of. The bit about Keybase comes from the disclosure policy that this replaces.

As I understand it, the SES group set up a Keybase team for security coordination some time ago. Without more information on the current state of the Keybase team, it seemed imprudent to remove the mention of it here and potentially close that avenue of disclosure without proactive messaging. If the Keybase team is deemed inactive or no longer desirable as a disclosure channel with further investigation, this can be removed/revised to reflect that.